audio_service_web/.gitignore

```diff
@@ -0,0 +1,7 @@
+.DS_Store
+.dart_tool/
+
+.packages
+.pub/
+
+build/
```

Comment:
I wonder if we can also ignore the .idea files. Actually, now that we have this directory structure, I was thinking of eventually moving some of these ignore rules into a global .gitignore file in the root directory of the repo, but for now we could just add the rule(s) here.

Reply:
We should also gut pubspec.lock in .gitignore